.php_cs

```diff
@@ -0,0 +1,13 @@
+<?php
+
+require_once './vendor/autoload.php';
+
+use Symfony\CS\Finder\DefaultFinder;
+use Symfony\CS\Config\Config;
+
+$finder = DefaultFinder::create()
+    ->in('src/');
+
+return Config::create()
```

Comment:
could you explicitly set the level to symfony here so the move to 2.x will be smooth